docs/make.jl

```diff
@@ -1,18 +1,78 @@
 using Documenter, Oscar, DocumenterMarkdown
+using AbstractAlgebra
+
+AA_dir = pkgdir(AbstractAlgebra)
+Oscar_dir = pkgdir(Oscar)
+
+cp(joinpath(AA_dir, "docs", "src"), joinpath(Oscar_dir, "docs", "src", "AAsrc"), force=true)
```

Comment:
just for my understanding: would a symlink perhaps also work?

Reply:
> just for my understanding: would a symlink perhaps also work?

I'm pretty sure yes, but I never bothered to understand how symlinks work on Windows, and the docstring of `symlink` says:

> This function raises an error under operating systems that do not support soft symbolic links, such as Windows XP.

Given how little work it is to copy the files here, I didn't think twice, but am happy to use `symlink`, or even `symlink` in `try-catch` followed by a copy, if you think it's worthwhile.